docs/references/release-changelogs.md

```diff
@@ -0,0 +1,39 @@
+# Release change-logs
+
+A clear and exhaustive [change-log](https://en.wikipedia.org/wiki/Changelog) is essential for customers who depend on the [`@azure/communication-react` package on NPM](https://www.npmjs.com/package/@azure/communication-react) to be able to update to newer versions of the package with confidence.
+
+Our change-log is mantained in-tree, rooted in the [top-level CHANGELOG.md file](https://github.com/Azure/communication-ui-library/blob/main/CHANGELOG.md).
+
+Because we separate [stable and beta release channels for the package](./beta-only-features.md), we need to maintain separate change-logs for customers on those channels. The top-level change-log links to these separate change-logs.
+
+At a high level, change-log maintenance happens in two steps:
+
+- Each change, in the form of a Pull Request to `main` in this repository adds a change-file - a file with a short description and type of the change.
+  - Developers generate these files by running `rush changelog` and CI enforces the existance of change files in Pull Requests.
+- The change-files are collected and summarized into the change-logs as part of the [release process](../releases/creating-a-release.md).
+
+## Under the hood
+
+Under the hood, our change-log tooling is built atop [beachball](https://microsoft.github.io/beachball/overview/getting-started.html). We do have a fair bit of customization / wrapping tooling to make it work with two release channels. The wrappers live in the [common/scripts/changelog/](../../common/scripts/changelog/) folder.
+
```

Comment:
Would be nice to add a example + a folder structure here for better demonstration (something like before after, for both collecting and changelog

for example:
before changelog command
```
- BETA_DIR/
  - changelog1.json
- CHANGELOG_DIR/
  - changelog1.json
```

```
- BETA_DIR/
  - changelog1.json
  - changelog2.json (new)
- CHANGELOG_DIR/
  - changelog1.json
  - changelog2.json (new)
```

Reply:
Also for before/after collecting

Reply:
It should be still working well without an example, but just feel that it took me some time to understand the logic when reading the doc + reading the code